examples/trusted_information_retrieval/config/config.toml

```diff
@@ -1 +1,9 @@
-database_url = "https://localhost:8888"
+name = "trusted_information_retrieval"
+
+[modules]
+app = { path = "examples/target/wasm32-unknown-unknown/release/trusted_information_retrieval.wasm" }
```

Comment:
I find it inelegant that this is referencing wasm modules via full target path. cargo has an experimental option to select an output folder for the binary. In #1214 I have out the config files in a "bin" folder. Maybe we should try the same with the wasm files?

Reply:
+1 for some sort of `out` / `bin` folder

Reply:
Will rewrite to use `bin` once #1214 is merged.

Reply:
Done